### PR TITLE
Template events for shop's cart

### PR DESCRIPTION
**Pull request by Tomanhez**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/ShopBundle/Resources/views/Cart/Block/_cartButton.html.twig [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{% import "@SyliusShop/Common/Macro/money.html.twig" as money %}
```
Comment (GSadee) [inline src/Sylius/Bundle/ShopBundle/Resources/views/Cart/Block/_cartButton.html.twig]: Why the name of this template is `_cartButton`? IMO you should add somewhere that this is a widget

File: src/Sylius/Bundle/ShopBundle/Resources/config/app/config.yml [hunk 2]
```diff
@@ -104,3 +104,108 @@ sylius_ui:
...
+                after_totals_legacy:
+                    template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    priority: 15
+                    context:
+                        event: sylius.shop.cart.summary.after_totals
+                checkout:
+                    template: "@SyliusShop/Cart/Block/_checkout.html.twig"
+                    priority: 10
+
+        sylius.shop.cart.widget.popup:
```
Comment (GSadee) [inline src/Sylius/Bundle/ShopBundle/Resources/config/app/config.yml]: This definition could be next to `sylius.shop.cart.widget.button`

File: src/Sylius/Bundle/ShopBundle/Resources/config/app/config.yml [hunk 3]
```diff
@@ -104,3 +104,108 @@ sylius_ui:
...
+                    template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    priority: 15
+                    context:
+                        event: sylius.shop.partial.cart.summary.before_widget_content
+                button:
+                    template: "@SyliusShop/Cart/Block/_cartButton.html.twig"
+                    priority: 10
+                after_widget_content_legacy:
+                    template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    priority: 15
```
Comment (Zales0123) [inline src/Sylius/Bundle/ShopBundle/Resources/config/app/config.yml]: `priority: 5` I suppose

File: src/Sylius/Bundle/ShopBundle/Resources/config/app/config.yml [hunk 4]
```diff
@@ -104,3 +104,108 @@ sylius_ui:
...
+                    context:
+                        event: sylius.shop.cart.summary.before_items
+                items:
+                    template: "@SyliusShop/Cart/Block/_items.html.twig"
+                    priority: 10
+                after_items_legacy:
+                    template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    priority: 15
```
Comment (Zales0123) [inline src/Sylius/Bundle/ShopBundle/Resources/config/app/config.yml]: Same here

File: src/Sylius/Bundle/ShopBundle/Resources/config/app/config.yml [hunk 5]
```diff
@@ -104,3 +104,109 @@ sylius_ui:
...
+                    priority: 20
+                after_totals_legacy:
+                    template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    priority: 15
+                    context:
+                        event: sylius.shop.cart.summary.after_totals
+                checkout:
+                    template: "@SyliusShop/Cart/Block/_checkout.html.twig"
+                    priority: 10
+
```
Comment (GSadee) [inline src/Sylius/Bundle/ShopBundle/Resources/config/app/config.yml]: Unnecessary blank line

**Comment by GSadee** [comment]
Thanks, Tomasz! :tada:

**Pull request merged**